### ISSUE TITLE
Tried withdrawing and collecting... but the transaction fails

### ISSUE THREAD

**Issue by GriffGreen** [Giveth/giveth-dapp#871]
<img width="1216" alt="Screen Shot 2019-07-06 at 12 13 08 AM" src="https://user-images.githubusercontent.com/14945613/60747049-f49cea80-9f82-11e9-9a9a-40325967cef8.png">

https://beta.giveth.io/campaigns/5cfe3366228b71296a491fa7/milestones/5d1ba874bd45530e23ad86ed

**Comment by GriffGreen** [comment]
never mind... it looks like it did work... but it sent me an error... i must have signed the metamask tx without thinking... so now i guess the issue is why it wasn't obvious that i had already clicked withdraw

**Comment by arbreton** [comment]
I agree, whenever an action is done there should be a UI feedback like graying out the buttons or a loading and if it fails it should also popup like on most cases, we can definetely look into this

**Comment by mroberts10** [comment]
Solution could be catch those error messsages from metamask and handling them through the DAPP.

**Comment by aminlatifi** [comment]
After trying several times, I couldn't reproduce the error. I just get error popup of DApp as it should be.
@GriffGreen I did collect transaction two times. Just ignored Metamask dialog after first hitting Withdraw button, did withdraw again and confirm two queue Metamask transactions. Is it the correct scenario?

**Comment by aminlatifi** [comment]
@GriffGreen  I am still trying to reproduce the situation. Dapp is running in `develop` mode. 
I captured my screen to show what was the scenario.
Please note donation record amount is shown 0.0000000000 Eth in some points. Maybe it's a bug!
[simplescreenrecorder-2019-07-18_11.38.17.zip](https://github.com/Giveth/giveth-dapp/files/3405223/simplescreenrecorder-2019-07-18_11.38.17.zip)

**Comment by GriffGreen** [comment]
oh man i'm sorry... i reopened an old issue cause i think i know the problem...

**Comment by GriffGreen** [comment]
^^^^ This is the issue... its cause there are too many small donations 🤦‍♂

**Comment by GriffGreen** [comment]
https://rinkeby.etherscan.io/tx/0x31008943b5575c004d8e095aa3e509026c62514970c62c87d5f42101b1d69105
boom baby

**Comment by GriffGreen** [comment]
I was able to get one batch out... but when trying to do the second batch it got too big again...

<img width="1344" alt="Screen Shot 2019-07-23 at 7 35 36 PM" src="https://user-images.githubusercontent.com/14945613/61733935-2aaedc80-ad81-11e9-9bb7-da601fecf5d0.png">


But wow, great message!!! :-D
<img width="593" alt="Screen Shot 2019-07-23 at 7 35 21 PM" src="https://user-images.githubusercontent.com/14945613/61733936-2b477300-ad81-11e9-87db-c55f3fca1b90.png">

**Comment by arbreton** [comment]
Reduced the total pledgeLimit to 8, that seems to be a safe area, it worked! that's awesome!

**Comment by arbreton** [comment]
Actually RJ already implemented but at some point it got increased to 18, and there is still some work to be done on the calculation of each one due this gas problems

**Comment by GriffGreen** [comment]
i think we might need to turn it down to 7

**Comment by arbreton** [comment]
I will hotfix that change!, sounds like a solution

**Comment by GriffGreen** [comment]
<img width="1324" alt="Screen Shot 2019-07-29 at 7 33 43 PM" src="https://user-images.githubusercontent.com/14945613/62069197-039c5300-b238-11e9-9665-0e891a1efb7c.png">
I wonder if there is a way to check what is going on when this is being dispersed....

**Comment by GriffGreen** [comment]
#621 is the real issue here

**Comment by arbreton** [comment]
second time withdrwaing

**Comment by arbreton** [comment]
it shows error

**Comment by GriffGreen** [comment]
https://beta.giveth.io/campaigns/5cfe3366228b71296a491fa7/milestones/5d1ba874bd45530e23ad86ed

Still doesnt work
:-(
What's the new direction? can we hack it?

**Comment by GriffGreen** [comment]
#621 is the issue
